nautobot/dcim/models/device_component_templates.py

```diff
@@ -80,6 +79,12 @@ def to_objectchange(self, action):
             object_data=serialize_object(self),
         )
 
+    def instantiate_return(self, model, **kwargs):
+        """Simplifies the instantiating of component on the specified Device by
+        predefining common arguments like name, label and description
+        """
+        return model(name=self.name, label=self.label, description=self.description, **kwargs)
```

Comment:
I like the idea here! Since all ComponentModels *must* have a `device`, let's make that explicit here, something like:

```suggestion
    def instantiate_model(self, model, device, **kwargs):
        """
        Helper method to self.instantiate().
        """
        return model(device=device, name=self.name, label=self.label, description=self.description, **kwargs)
```